File: Java/TP3-4/src/Group.java
public class Group extends Usuario {

	public Group(String login) {
		super(login);
	}

	public void despedidaGroup() {
		System.out.println("Log-off " + super.getLogin());
	}

	public void descricaoGroup() {
		System.out.println("Pode participar de Grupos.");
	}

	public void groupBoard() {
		System.out.println("Você remover Usuários deste Grupo");
	}
}

File: Kotlin/TPK3-4/bin/Group.kt
class Group(login:String):Usuario(login) {
  fun despedidaGroup() {
    System.out.println("Log-off " + super.getLogin())
  }
  fun descricaoGroup() {
    println("Pode participar de Grupos.")
  }
  fun groupBoard() {
    println("Você remover Usuários deste Grupo")
  }
}